### DISCUSSION TITLE
There is no information about the languages used in the project

### DISCUSSION THREAD

**Discussion by olekstomek** [community/community#34651] [Mobile]
On desktop version I can see used languages inside repository:
![image](https://user-images.githubusercontent.com/26818304/193146257-f95b3eab-51bb-4e2f-b269-89b93baa2fe2.png)
This would be very useful in a mobile application.

**Comment by radar-blip** [comment]
The mobile app doesn't have all the features of the desktop version.
I don't know if you were just making a suggestion, but that's the reason for this.

**Comment by bogedy** [comment]
Just bumping this to say that I find this feature essential for figuring out what a repo actually is. Have to uninstall the app for now in order to force links to open in the browser.

**Comment by 03timothyash** [comment]
Así es todo va bien y dependerá de la base crunchbase

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
